completion/models.py

```diff
@@ -168,6 +168,38 @@ class BlockCompletion(TimeStampedModel, models.Model):
 
     objects = BlockCompletionManager()
 
+    @classmethod
+    def get_course_completions(cls, user, course_key):
+        """
+        query all completions for course/user pair
```

Comment:
This could be a little more descriptive: "Returns a dictionary mapping BlockKeys to completion values for all BlockCompletion records for the given user and course_key."

Reply:
That's fair.